packages/Operators/src/Factory/DTK_PointCloudOperatorFactory.hpp

```diff
@@ -90,7 +90,8 @@ class PointCloudOperatorFactory
     enum PointCloudOperatorType
     {
 	SPLINE_INTERPOLATION,
-	MOVING_LEAST_SQUARE
+	MOVING_LEAST_SQUARE,
+        NODE_TO_NODE
     };
```

Comment:
these are only suggestions for improvement and don't need to be addressed in this PR.